core/services/pipeline/task.base.go

```diff
@@ -6,10 +6,11 @@ type BaseTask struct {
 	outputs []Task
 	inputs  []Task
 
-	id      int
-	dotID   string
-	Index   int32         `mapstructure:"index" json:"-" `
-	Timeout time.Duration `mapstructure:"timeout"`
+	id        int
+	dotID     string
+	Index     int32         `mapstructure:"index" json:"-" `
+	Timeout   time.Duration `mapstructure:"timeout"`
+	FailEarly string        `mapstructure:"failEarly"`
```

Comment:
@spooktheducks I wasn't able to parse to `bool` directly here, `UnmarshalTaskFromMap` would fail.

Reply:
Can we get some documentation on this? Add it to: https://github.com/smartcontractkit/documentation/pull/229/files